src/utils/testFixtures.js

```diff
@@ -1,4 +1,6 @@
 /* @flow */
+/* eslint-disable */
```

Comment:
do you need to disable all the rules?

Reply:
yes otherwise it will complain over magic numbers and such (ids in arrays)

Reply:
how many rules is it complaining about?

If it's less than 5 I'd prefer it if you turned off just those rules

Reply:
its just magic numbers, I can just disable that rule and then we get to keep all the others. I think we should at least have those arrays of ids for testing

Reply:
Yes, I totally agree that we should have these tests.

I'm just reluctant to nuke all the lint rules at once